src/condition/sqli_detector.hpp

```diff
@@ -13,6 +13,7 @@ namespace ddwaf {
 
 class sqli_detector : public base_impl<sqli_detector> {
 public:
+    static constexpr unsigned version = 2;
```

Comment:
Out of interest. Why is this v2 and not v1?

Reply:
We made some changes to the heuristic recently so this one is now v2 to distinguish it from the one released previously.